### ISSUE TITLE
cinnamon: Network window is very little

### ISSUE THREAD

**Issue by kacperpaczos** [linuxmint/mint19.2-beta#49]
**Describe the bug**
When I open network window, windows is too short.

**To Reproduce**
Open network settings by network applet.
![obraz](https://user-images.githubusercontent.com/23406555/61581303-dc010880-ab1c-11e9-8f85-d165de7005e1.png)

**Expected behavior**
Windows should be normal height.
**Frequency**
Does the issue happen:
- [*] Always
- [ ] Randomly

**Edition (Desktop environment):**
In which edition is this happening?
- [*] Cinnamon Edition
- [ ] MATE Edition
- [ ] Xfce Edition

**Live or post-install:**
Indicate if this is happening after the installation, or during the live session:
- [*] Post-install
- [ ] Live session

**Screenshots**
![obraz](https://user-images.githubusercontent.com/23406555/61581267-65640b00-ab1c-11e9-9dc5-1f4b1cdbbd6f.png)

**Locale:**
Paste the output of `locale`.

**Drivers:**
Paste the output of `inxi -Gxx`.

**Crash report:**

''Stack trace''
If you are reporting a crash. Go to "Software Sources" and enable the debug symbols. Then go to "System Reports" and extract the stack trace for your crash and paste it here.

''Dmesg''
If you type `dmesg` in a terminal, you should see a line corresponding to the crash. Paste it here.

**Additional context**
If you open network window in control center, window height is normal.

**Comment by clefebvre** [comment]
I can't reproduce... do you have some resolution info, a complete screenshot?

**Comment by Rad71** [comment]
Confirmed. Reproduce on Mint 19.1
Graphics:
  Device-1: Intel Atom Processor Z36xxx/Z37xxx Series Graphics & Display 
  driver: i915 v: kernel 
  Display: x11 server: X.Org 1.19.6 driver: modesetting unloaded: fbdev,vesa 
  resolution: 1366x768~60Hz 
  OpenGL: renderer: Mesa DRI Intel Bay Trail v: 4.2 Mesa 19.0.2

**Comment by clefebvre** [comment]
![image](https://user-images.githubusercontent.com/1138515/61582227-48353980-ab28-11e9-985b-d6835071e7dd.png)

This is it in 1024x768... how is it too short?

**Comment by Rad71** [comment]
![obraz](https://user-images.githubusercontent.com/53124039/61582335-d1993b80-ab29-11e9-91bc-d235f6ae517c.png)
That's how it opens

**Comment by clefebvre** [comment]
what is your locale?

**Comment by clefebvre** [comment]
does it happen when you run cinnamon-settings, and then go to network? and when you run "cinnamon-control-center"?

**Comment by Rad71** [comment]
LANG=pl_PL.UTF-8
LANGUAGE=pl
LC_CTYPE="pl_PL.UTF-8"
LC_NUMERIC="pl_PL.UTF-8"
LC_TIME="pl_PL.UTF-8"
LC_COLLATE="pl_PL.UTF-8"
LC_MONETARY="pl_PL.UTF-8"
LC_MESSAGES="pl_PL.UTF-8"
LC_PAPER="pl_PL.UTF-8"
LC_NAME="pl_PL.UTF-8"
LC_ADDRESS="pl_PL.UTF-8"
LC_TELEPHONE="pl_PL.UTF-8"
LC_MEASUREMENT="pl_PL.UTF-8"
LC_IDENTIFICATION="pl_PL.UTF-8"
LC_ALL=

pyroxar I see has the same locale.

**Comment by Rad71** [comment]
![obraz](https://user-images.githubusercontent.com/53124039/61582588-dd3a3180-ab2c-11e9-8b6b-1dbb0546b7d4.png)

Only open this way. (Picture from the problem submission from pyroxar).
EDIT:
From menu is identically.  (Menu->Preferences->Network)
From control-center is ok.

**Comment by Rad71** [comment]
From control-center is ok.

**Comment by pn-santos** [comment]
This happens in 19.1 too btw. Exactly as described:

> From menu is identically. (Menu->Preferences->Network)
> From control-center is ok.

```
System:    Host: Y01130 Kernel: 4.15.0-54-generic x86_64 bits: 64 Desktop: Cinnamon 4.0.10 Distro: Linux Mint 19.1 Tessa 
Machine:   Type: Laptop System: Dell product: Latitude E7470 v: N/A serial: <root required> 
           Mobo: Dell model: 0T6HHJ v: A00 serial: <root required> UEFI [Legacy]: Dell v: 1.21.6 date: 05/20/2019 
Battery:   ID-1: BAT0 charge: 52.2 Wh condition: 52.2/55.0 Wh (95%) 
CPU:       Topology: Dual Core model: Intel Core i7-6600U bits: 64 type: MT MCP L2 cache: 4096 KiB 
           Speed: 500 MHz min/max: 400/3400 MHz Core speeds (MHz): 1: 500 2: 500 3: 500 4: 500 
Graphics:  Device-1: Intel Skylake GT2 [HD Graphics 520] driver: i915 v: kernel 
           Display: x11 server: X.Org 1.19.6 driver: modesetting unloaded: fbdev,vesa resolution: 1920x1080~60Hz 
           OpenGL: renderer: Mesa DRI Intel HD Graphics 520 (Skylake GT2) v: 4.5 Mesa 19.0.2 
Audio:     Device-1: Intel Sunrise Point-LP HD Audio driver: snd_hda_intel 
           Sound Server: ALSA v: k4.15.0-54-generic 
Network:   Device-1: Intel Ethernet I219-LM driver: e1000e 
           IF: enp0s31f6 state: down mac: 18:db:f2:06:68:da 
           Device-2: Intel Wireless 8260 driver: iwlwifi 
           IF: wlp1s0 state: up mac: e4:a7:a0:75:c0:9c 
           IP v4: 192.168.1.6/24 type: dynamic noprefixroute scope: global 
           IP v6: fe80::3275:30d2:4f6f:be13/64 type: noprefixroute scope: link 
           IF-ID-1: docker0 state: down mac: 02:42:ab:55:34:df 
           IP v4: 172.17.42.1/24 scope: global 
           IP v6: fe80::42:abff:fe55:34df/64 scope: link 
           IF-ID-2: tun0 state: down mac: N/A 
           WAN IP: No WAN IP data found. Connected to the web? SSL issues? 
Drives:    Local Storage: total: 238.47 GiB used: 62.49 GiB (26.2%) 
           ID-1: /dev/sda vendor: Toshiba model: THNSNK256GVN8 M.2 2280 256GB size: 238.47 GiB 
RAID:      Hardware-1: Intel 82801 Mobile SATA Controller [RAID mode] driver: ahci 
Partition: ID-1: / size: 226.54 GiB used: 62.39 GiB (27.5%) fs: ext4 dev: /dev/dm-1 
           ID-2: /boot size: 471.6 MiB used: 80.7 MiB (17.1%) fs: ext2 dev: /dev/sda1 
           ID-3: swap-1 size: 7.67 GiB used: 31.2 MiB (0.4%) fs: swap dev: /dev/dm-2 
Sensors:   System Temperatures: cpu: 46.0 C mobo: N/A 
           Fan Speeds (RPM): cpu: 3293 
Info:      Processes: 289 Uptime: 4d 11h 04m Memory: 15.49 GiB used: 9.82 GiB (63.4%) Shell: zsh inxi: 3.0.27
```

**Comment by clefebvre** [comment]
@pn-santos what's your locale?

**Comment by MinIsMin** [comment]
Can confirm this on 19.1 with locale de_DE.UTF-8 on display 1920x1080

**Comment by pn-santos** [comment]
@clefebvre it's `en_GB.utf-8`

**Comment by smurphos** [comment]
For Cinnamon 4.0.x / 19.1 this bug was reported here https://github.com/linuxmint/cinnamon/issues/8360  and should be fixed in the beta with Cinnamon 4.2. I've not near my machine with the beta installed at the moment to check.

Edit to add: I can't reproduce in the beta, only in 19.1.

**Comment by clefebvre** [comment]
Closing as this looks like it was fixed.
